module/Decision/src/Decision/Service/Member.php

```diff
@@ -87,7 +87,8 @@ public function findMemberByLidNr($lidnr)
      */
     public function getBirthdayMembers($days = 0)
     {
-        if (!$this->isAllowed('birthdays')) {
+        // Allow everyone to view birthdays of today.
+        if ($days > 0 && !$this->isAllowed('birthdays')) {
```

Comment:
You should actually add a privilege for showing today's birthdays, and check for that privilige for the case of `$days == 0`